lib/maps/thermostat.js

```diff
@@ -6,7 +6,11 @@ module.exports = (Mapper, Service, Characteristic) => ({
       name: 'fan',
       overrides: {
         required: {},
-      },
+        // Avoid conflict with thermostat on/off
```

Comment:
After testing some days, this seems needed, otherwise the fan speed state can appear 20 % (written value on the top) but the slider appears 0%.